public/content/translations/zh/community/online/index.md

```diff
@@ -25,12 +25,12 @@ lang: zh
 <SocialListItem socialIcon="discord"><Link to="https://ethglobal.co/discord">Ethereum Hackers</Link> - 由 ETHGlobal 管理的 Discord 聊天室：全世界以太坊黑客的线上社区。</SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/5W5tVb3">CryptoDevs</Link> - 关注以太坊发展的 Discord 社区</SocialListItem>
 <SocialListItem socialIcon="discord"><Link to="https://discord.gg/ethstaker">EthStaker Discord</Link> - 社区为现有和潜在的质押人提供的指导、教育、支持和资源。</SocialListItem>
-<SocialListItem socialIcon="discord"><Link to="https://discord.gg/ethereum-org">Ethereum.org 网站团队</Link> - 拜访社区中的团队和成员并与他们讨论 ethereum.org 的网络开发和设计</SocialListItem>
+<SocialListItem socialIcon="discord"><Link to="https://discord.gg/CetY6Y4">Ethereum.org 网站团队</Link> - 拜访社区中的团队和成员并与他们讨论 ethereum.org 的网络开发和设计</SocialListItem>
```

Comment:
Not sure why this flipped back, but `https://discord.gg/ethereum-org` is used in canonical English file and in most other places on the site. Flipping back

```suggestion
<SocialListItem socialIcon="discord"><Link to="https://discord.gg/ethereum-org">Ethereum.org 网站团队</Link> - 拜访社区中的团队和成员并与他们讨论 ethereum.org 的网络开发和设计</SocialListItem>
```